dev/src/Command/AddComponentCommand.php

```diff
@@ -201,4 +242,73 @@ private function loadProtoContent(string $proto): string
         $response = $client->get($protoUrl);
         return (string) $response->getBody();
     }
+
+    private function bazelBuildLibrary(string $libraryName, string $googleApisDir): bool
+    {
+        $command = 'which bazel';
+        $output = exec($command, $output, $return);
+        if (strlen($output) == 0) {
+            exit("Error: Bazel is not installed. Please install Bazel before running this script.\n");
+        }
+        chdir($googleApisDir);
+        $command = "bazel query 'filter(\"-(php)$\", kind(\"rule\", //google/cloud/" . $libraryName .
+            "/...:*))' | grep -v -E \":(proto|grpc|gapic)-.*-php$\"";
+        exec($command, $output, $return);
+        if ($return === 0 && !empty($output)) {
+            $command = "bazel build " . $output[0];
+            exec($command, $output, $return);
+            if ($return === 0) {
+                chdir($this->rootPath);
+                return true;
+            }
+        }
+        chdir($this->rootPath);
+        exit("Something went wrong at bazelBuildLibrary, Please use the add-component command instead.\n");
+    }
+
+    private function copyBazelOutToGoogleCloudPhp(string $componentName, string $googleApisDir): bool
+    {
+        $command = 'which docker';
+        $output = exec($command, $output, $return);
+        if (strlen($output) == 0) {
+            exit("Error: Docker is not installed. Please install Docker before running this script.\n");
+        }
+        $googleApisDir = realpath($googleApisDir);
+        $owlbotImage = "gcr.io/cloud-devrel-public-resources/owlbot-cli:latest";
+        $command = "docker run --rm --user $(id -u):$(id -g) -v " . $this->rootPath .
+        ":/repo -v " . $googleApisDir . "/bazel-bin:/bazel-bin " . $owlbotImage .
+        " copy-bazel-bin --config-file=" . $componentName . "/.OwlBot.yaml --source-dir /bazel-bin --dest /repo >&2";
+        exec($command, $output, $return);
+        if ($return === 0) {
+            return true;
+        }
+        exit("Something went wrong at copyBazelOutToGoogleCloudPhp, Please use the add-component command instead.\n");
+    }
+
+    private function postProcess()
+    {
+        $command = "docker run --user $(id -u):$(id -g) --rm " .
+        "-v " . $this->rootPath .":/repo " .
+        "-w /repo gcr.io/cloud-devrel-public-resources/owlbot-php >&2";
+        exec($command, $output, $return);
+        if ($return === 0) {
+            return true;
+        }
+        exit("Something went wrong at postProcess, Please use the add-component command instead.\n");
+    }
+
+    private function askConfirmationAboutLogs(InputInterface $input, OutputInterface $output, string $step)
+    {
+        $output->writeln('');
+        if (!$this->getHelper('question')->ask(
+            $input,
+            $output,
+            new ConfirmationQuestion(sprintf('Does the logs look good for %s [Y/n] ', $step), 'Y')
```

Comment:
To me, it's a bit confusing to me to say "does the logs look good".

If you want confirmation, you could do something like add this to a `/tmp` file and say "Please confirm that we will add the following files" or something like that. Or just say "We added the following files...".

But maybe a better question is why we think a confirmation is needed at all? I am not opposed to it, but it seems a bit silly to prompt the developer 3 times when they only say yes or no. Since they should be confirming the changes in the git commit before submitting the pull request, and the pull request during code review, anyway.

Reply:
Yeah, sounds good, we can remove these confirmations then.

Reply:
Done!